server/src/lib/utils/validator.ts

```diff
@@ -92,7 +107,7 @@ export function validateJson(
     finalErrors = validateSlug(data.slug,uniqueSlugs,validate.errors || [])
   }
   if(data.body) {
-    finalErrors = validateBody(data.body, validate.errors || [])
+    finalErrors = finalErrors.concat(validateBody(data.body, validate.errors || []));
```

Comment:
why do we need the `concat`?  `validateBody` already returns the updated array right?